packages/yuudachi/locales/en-US/translation.json

```diff
@@ -381,13 +381,17 @@
 				"embeds": "• Embeds: {{- embeds}}",
 				"attachments": "• Attachments: {{- attachments}}",
 				"stickers": "• Stickers: {{- stickers}}",
-				"jump_to": "• [Jump to]({{- link}})"
+				"jump_to": "• [Jump to]({{- link}})",
+				"reply_to": "• Replying to [{{message_id}}]({{- message_url}}) by `{{- user_tag}}` ({{- user_id}})",
+				"reply_to_mentions": "• @Replying to [{{message_id}}]({{- message_url}}) by `{{- user_tag}}` ({{- user_id}})"
```

Comment:
`user_id` doesn't need the `-` since nothing will be escaped in a numbered string.